### PR TITLE
release: v0.0.31

### PR DESCRIPTION
**Pull request by salguarnieri**
https://github.com/kythe/kythe/releases/edit/untagged-bcc01fab3bf3d8398340

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -1,5 +1,321 @@
...
+  *  add build config to each related Anchor (#3738) ([b8f01f6d](https://github.com/kythe/kythe/commit/b8f01f6d4ae9e4ea3d530014f50359142791a692))
+  *  allow a semantic scope to be associated per decoration (#3707) ([643cc854](https://github.com/kythe/kythe/commit/643cc854586ce4db4940ab7ab8fb068c7e44fcff))
+  *  add set of build configs per Corpus in CorpusRootsReply (#3572) ([96cb400a](https://github.com/kythe/kythe/commit/96cb400a36dfa146d91fb98fad75ba66a24d1aaa))
+  *  add build configurations per FileTree entry (#3523) ([b5eddd52](https://github.com/kythe/kythe/commit/b5eddd52caa6251e6543b6b6be1f0ad43784e27f))
+  *  add CrossReferencesRequest filter for build config (#3512) ([cc9d2d66](https://github.com/kythe/kythe/commit/cc9d2d6650ef83832d8bbdaf86f2c6c33b8efbb6))
+* **bazel:**
+  *  support package installation at extraction time to support system dependencies (#3641) ([e7d15468](https://github.com/kythe/kythe/commit/e7d15468b396e7a3926b2c601984a0ca678ecbda))
+  *  Use bazelisk in Bazel extractor Dockerfile extraction. (#3596) ([94e70cde](https://github.com/kythe/kythe/commit/94e70cde56a6f2c2413a47a4dc125a5f35dd94fb))
+* **build:**
```
Comment (schroederc) [inline RELEASES.md]: `build` features aren't very relevant for the release notes.

File: RELEASES.md [hunk 2]
```diff
@@ -1,5 +1,321 @@
...
+  *  add property/* edges for @AutoValue classes (#3993) ([0abf28c9](https://github.com/kythe/kythe/commit/0abf28c9123a853edff350c1b4d4c254619006c0))
+  *  add basic tests for java-9-10-11 features (#3898) ([2139771c](https://github.com/kythe/kythe/commit/2139771cc23b12f93f8d439d349ca6dbc9d1e5ec))
+  *  emit refs to JVM graph for externally defined nodes (#3800) ([0d7614ed](https://github.com/kythe/kythe/commit/0d7614eda17a0d659b120d6479965d73ad861093))
+  *  add receiver type to function types (#3628) ([fe7aa759](https://github.com/kythe/kythe/commit/fe7aa7596bedb248ec6f8f0cd440a23d8eafbdc9))
+  *  add MarkedSources for function tapp nodes (#3601) ([f36eceff](https://github.com/kythe/kythe/commit/f36eceff4dc4376aed1ed5bcaa4bfc038a81aa6a))
+  *  add MarkedSources for array/generic tapp nodes (#3561) ([e2aaf23c](https://github.com/kythe/kythe/commit/e2aaf23c788f781bfdd2b3a2a30274d920192c73))
+* **jdk_indexing:**  allow extracting only specific make targets (#3935) ([2978be61](https://github.com/kythe/kythe/commit/2978be6173622481b2761a9ea4869f54b2a1fbdf))
+* **kzip:**
```
Comment (schroederc) [inline RELEASES.md]: All the `kzip` stuff is probably better situated under the `tooling` features.

File: RELEASES.md [hunk 3]
```diff
@@ -1,5 +1,321 @@
...
+  *  allow embedding details messages in kzips (#3538) ([d0255f56](https://github.com/kythe/kythe/commit/d0255f56a0d6336469aff463ff032d40f0d30176))
+* **post_processing:**
+  *  support Riegeli Snappy compression (#4072) ([a7631889](https://github.com/kythe/kythe/commit/a76318890a9373be509f3478f946c703dc60e175))
+  *  add Nodes.fromSource conversion utility (#3755) ([03bdb570](https://github.com/kythe/kythe/commit/03bdb57052655c8c352ccb6dfdd82ad33946b474))
+  *  allow nodes/anchors to be ranked (#3750) ([466b7bc4](https://github.com/kythe/kythe/commit/466b7bc4e4642b953c6285264cdf31f11f0f8686))
+  *  implement basic callers in PagedCrossReferences (#3712) ([87f273d1](https://github.com/kythe/kythe/commit/87f273d184cae9d47684cb6a58e95ef842e82ea8))
+  *  support reading input files from a directory (#3497) ([e907832f](https://github.com/kythe/kythe/commit/e907832f605731117ff5cd0b92c3e582806512e5))
+* **proto:**
+  *  add indexer for text-format protobuf (#39) ([d9c96147](https://github.com/kythe/kythe/commit/d9c96147cfb2affeaec4fb39e905033d3ae8f10d))
```
Comment (schroederc) [inline RELEASES.md]: Add to `textproto_indexer`

File: RELEASES.md [hunk 4]
```diff
@@ -1,5 +1,321 @@
...
+  *  empty deprecation message and additional test case (#3678) ([d56bc1d3](https://github.com/kythe/kythe/commit/d56bc1d3f43badb99c313288c50aa8109b2e141d))
+  *  skip synthesized map entry types (#26) ([c23d4d52](https://github.com/kythe/kythe/commit/c23d4d5271cb22174328a6b77be6086c7dd55c22))
+  *  close file after use in SourceTree.Read() (#18) ([da1c9cc0](https://github.com/kythe/kythe/commit/da1c9cc0779b92748249be34d2d455934f6943a1))
+  *  add missing dependency (#11) ([d534eedb](https://github.com/kythe/kythe/commit/d534eedba09d341bcc6f45725074d7519640934a))
+* **serving:**
+  *  properly return lookup errors (#3764) ([d5c0e4ec](https://github.com/kythe/kythe/commit/d5c0e4ec7a5505f3f63a76a1ec53a860319163b0))
+* **tests:**
+  *  make proto/textproto tests not rely on `jq` (#3967) ([28986284](https://github.com/kythe/kythe/commit/289862842f3c08317f531d0f064e8880a22b321b))
+  *  remove deadline option from proto testdata (#8) ([7f2f889d](https://github.com/kythe/kythe/commit/7f2f889d3409feb2bec7d73ffe5b3ac2647f89c0))
+* **textproto:**
```
Comment (schroederc) [inline RELEASES.md]: `textproto_indexing`

File: RELEASES.md [hunk 5]
```diff
@@ -1,5 +1,321 @@
...
+  *  Do not name scopes created by rvalues (#3841) ([4e6d241f](https://github.com/kythe/kythe/commit/4e6d241f8319c5a31c8d4e8d74bafe1b35cd0bc5), closes [#3834](https://github.com/kythe/kythe/issues/3834))
+  *  differentiate signatures of static and instance members (#3819) ([74e1e4a2](https://github.com/kythe/kythe/commit/74e1e4a2771a7c58ddbd56a00fcf0505b6c082a1))
+  *  point class type and values at the class name (#3835) ([35f41ccd](https://github.com/kythe/kythe/commit/35f41ccd0cc1d789f811de672fa68f097f8d4483))
+  *  remove deprecated childof edge from anchor to file (#3724) ([5533e861](https://github.com/kythe/kythe/commit/5533e8612cb0913906c45345c71bdd290296e635))
+  *  properly lookup file VNames (#3686) ([34369679](https://github.com/kythe/kythe/commit/3436967916f58850d7b837230d04207c08e1d51d))
+  *  obey VNames that come in from the input (#3467) ([a2bc9861](https://github.com/kythe/kythe/commit/a2bc9861d46403a47e38ae6506bba4ce7c98e0e2))
+* **vname_util:**  add missing #includes (#12) ([d2ff3cc5](https://github.com/kythe/kythe/commit/d2ff3cc57acf7ecf2ed5f41b028a46ee768de92b))
```
Comment (schroederc) [inline RELEASES.md]: Not relevant from last release.

File: RELEASES.md [hunk 6]
```diff
@@ -1,5 +1,321 @@
...
+* **jdk_extractor:**  bazel runfiles no longer expect the workspace name (#4131) ([ae19c938](https://github.com/kythe/kythe/commit/ae19c938a79218865f9dd320865397f5a15d05d9))
+* **kzip:**
+  *  populate KzipInfo.total_files in info.go (#4018) ([d1b99218](https://github.com/kythe/kythe/commit/d1b9921820b8992a0157473e319eb0bccf5dfd11))
+  *  make default kzip encodings explicit (#4012) ([b526e547](https://github.com/kythe/kythe/commit/b526e5474ef2185a9081f20dacfab1c8d19a182f))
+  *  use proper incantation so testdata still found on import (#3846) ([06f9aa46](https://github.com/kythe/kythe/commit/06f9aa46fac3adc65f732bc8fad898782c29e9cd))
+  *  EncodingFlag fully implements flag.Getter and flag.Value (#3843) ([55cf0dd2](https://github.com/kythe/kythe/commit/55cf0dd2a69840d0a66a4057c31f37957e8cf82a))
+* **kzip info:**  determine unit corpus and file language more robustly (#4095) ([c4a354e8](https://github.com/kythe/kythe/commit/c4a354e835acef2b9eb14d026142f440b5871fd0))
+* **kzip_create:**  implement flag.Getter for all flat types (#3544) ([a47b32ed](https://github.com/kythe/kythe/commit/a47b32ed1f6a747a0606cad38ac30340386a056a))
+* **leiningen:**
```
Comment (schroederc) [inline RELEASES.md]: Not relevant for release.

File: RELEASES.md [hunk 7]
```diff
@@ -1,5 +1,321 @@
...
+  *  emit and test type parameter MarkedSources (#3832) ([7cb46e3f](https://github.com/kythe/kythe/commit/7cb46e3f95b7795a4b597db48bda41a1290a5815))
+  *  emit MarkedSource for defined nodes referenced earlier in analysis (#3826) ([e3d031cc](https://github.com/kythe/kythe/commit/e3d031cce4f5e7e5c1ef38aa5078abc8b4726702))
+  *  avoid NPE when referencing Java package in comment (#3803) ([42f94049](https://github.com/kythe/kythe/commit/42f940495360beb4f83f9e08c365941b2a4e0bc7))
+  *  only emit MarkedSource for defined symbols (#3792) ([3079f619](https://github.com/kythe/kythe/commit/3079f6199e30f34cc7911af56cb9e79f0ef1bb1c))
+  *  only emit MarkedSource for defined symbols (#3788) ([6bfb2f7a](https://github.com/kythe/kythe/commit/6bfb2f7a46e7704f25073b7f419c34062218765a))
+  *  box LOOKUP_BY_PARAM MarkedSources with text (#3632) ([44771993](https://github.com/kythe/kythe/commit/44771993f97a7bd61b64a75884595ea3979baead))
+* **javac-wrapper:**  use `set -e` to propagate errors (#4042) ([f37f1c90](https://github.com/kythe/kythe/commit/f37f1c90e1e1a6182edcb595f62362f631a8a0ea))
```
Comment (schroederc) [inline RELEASES.md]: Remove.  This was reverted

File: RELEASES.md [hunk 8]
```diff
@@ -1,5 +1,321 @@
...
+  *  allow resolving relative paths against CU working dir (#3937) ([8050ceec](https://github.com/kythe/kythe/commit/8050ceeca656443266885c879f0afb635869d452))
+  *  replace or remove JDK9+ methods (#3930) ([b1810d84](https://github.com/kythe/kythe/commit/b1810d84044a8d654c299b8f27e80eb0ae837be6))
+* **java_extractor:**
+  *  include working directory in extracted proto (#4130) ([bc363f87](https://github.com/kythe/kythe/commit/bc363f877d35263dcbd0112d93a443e66481d2c1))
+  *  and tests should use ambient langtools (#4053) ([7f680e94](https://github.com/kythe/kythe/commit/7f680e94bb60ca23219421f6479dbf1c63a8f37b))
+  *  support missing -s in aspect extraction (#3986) ([7609b970](https://github.com/kythe/kythe/commit/7609b9700e6a30a5ec052e9baa713e59f073bee0))
+  *  deal with both kinds of Generated annotations (#3889) ([048134fd](https://github.com/kythe/kythe/commit/048134fd919bd263d996c22c226df70955bf23c7))
+  *  change an iterable to a collection (#3620) ([0daf616c](https://github.com/kythe/kythe/commit/0daf616c191abcc788a12b1915d8e8e45ba2cce7))
```
Comment (schroederc) [inline RELEASES.md]: Not relevant since last release.

File: RELEASES.md [hunk 9]
```diff
@@ -1,5 +1,321 @@
...
+  *  move JDK9 method reflection to static members (#3616) ([f4373076](https://github.com/kythe/kythe/commit/f4373076d655944e6f0853e96f775461e81e46cb))
```
Comment (schroederc) [inline RELEASES.md]: Not relevant since last release or a bug fix

File: RELEASES.md [hunk 10]
```diff
@@ -1,5 +1,321 @@
...
+  *  use reflection to access/override JDK bits. (#3612) ([72915498](https://github.com/kythe/kythe/commit/72915498ba2b803b180543db35bcfa894a2c4c03))
+  *  initial work enabling extraction of openjdk11 (#3607) ([3ad566cd](https://github.com/kythe/kythe/commit/3ad566cd3cf258c38a2d72062d30be26932d9092))
```
Comment (schroederc) [inline RELEASES.md]: Not a bug fix

File: RELEASES.md [hunk 11]
```diff
@@ -1,5 +1,321 @@
...
+  *  minor java changes as suggested by downstream analyzers (#3860) ([7a9ed0ec](https://github.com/kythe/kythe/commit/7a9ed0ec6aee13c8fba1ccf8d4581939805d21d2))
+  *  accept either JRE 8 or JRE 9 style classpath in test (#3972) ([3ea7de6c](https://github.com/kythe/kythe/commit/3ea7de6ccd55e30a693e60996468a14f78151630))
+* **java_build:**  remove jvm_flags which complicate running on newer JDKs (#3955) ([b4bdb395](https://github.com/kythe/kythe/commit/b4bdb395bd108415e5d8deab65a6538915d31a6c))
+* **java_common:**
+  *  normalize paths before looking them up (#4106) ([d4155894](https://github.com/kythe/kythe/commit/d4155894a82c556742250a17868abfb16625ef26))
+  *  adjust case when getting default format (#4104) ([64659661](https://github.com/kythe/kythe/commit/646596615567fb3d63d5d50c29bdfe9896cd4524))
+  *  cufs shares FileDataProvider and should not close it (#4103) ([0692beab](https://github.com/kythe/kythe/commit/0692beab05f74f201c9f3112d1df0b3bf34f626b))
+  *  mark CompilationUnitFileSystem public (#3977) ([2d72afe9](https://github.com/kythe/kythe/commit/2d72afe9170521f62a08cbb907018d000a60394a))
```
Comment (schroederc) [inline RELEASES.md]: Not relevant since last release.

File: RELEASES.md [hunk 12]
```diff
@@ -1,5 +1,321 @@
...
+* **java:**
+  *  handle more null data from java (#4082) ([e878b5f5](https://github.com/kythe/kythe/commit/e878b5f58a8a7174b2c83373b89f60b99ce9b092))
+  *  minor java changes as suggested by downstream analyzers (#3860) ([7a9ed0ec](https://github.com/kythe/kythe/commit/7a9ed0ec6aee13c8fba1ccf8d4581939805d21d2))
+  *  accept either JRE 8 or JRE 9 style classpath in test (#3972) ([3ea7de6c](https://github.com/kythe/kythe/commit/3ea7de6ccd55e30a693e60996468a14f78151630))
+* **java_build:**  remove jvm_flags which complicate running on newer JDKs (#3955) ([b4bdb395](https://github.com/kythe/kythe/commit/b4bdb395bd108415e5d8deab65a6538915d31a6c))
+* **java_common:**
+  *  normalize paths before looking them up (#4106) ([d4155894](https://github.com/kythe/kythe/commit/d4155894a82c556742250a17868abfb16625ef26))
+  *  adjust case when getting default format (#4104) ([64659661](https://github.com/kythe/kythe/commit/646596615567fb3d63d5d50c29bdfe9896cd4524))
+  *  cufs shares FileDataProvider and should not close it (#4103) ([0692beab](https://github.com/kythe/kythe/commit/0692beab05f74f201c9f3112d1df0b3bf34f626b))
```
Comment (schroederc) [inline RELEASES.md]: Not relevant since last release.

File: RELEASES.md [hunk 13]
```diff
@@ -1,5 +1,321 @@
...
+  *  -tags is space-separated, not comma (#3856) ([c932970f](https://github.com/kythe/kythe/commit/c932970f0e11b0f8198a137535bc80a7ffe8341a))
+* **go extractor:**  fix invalid Details messages for packages (#4120) ([426a9b0b](https://github.com/kythe/kythe/commit/426a9b0be61ff6eb35a9bc94fdd4e73d79e28138))
+* **go_common:**  use nondeprecated zip file handle mod time (#4047) ([b9e9c011](https://github.com/kythe/kythe/commit/b9e9c01155d91df10833c2faafe769f35f9b6aba))
+* **go_indexer:**
+  *  use only name for a named type's MarkedSource (#3850) ([f363e506](https://github.com/kythe/kythe/commit/f363e50690b65727ba33a8e09a57b177788b24db))
+  *  handle overrides for structs in a other packages (#3606) ([f3364504](https://github.com/kythe/kythe/commit/f3364504337e26e5a407a63c9c6f947b22b72cb3))
+  *  always emit anchor for anon value types (#3605) ([d97ca443](https://github.com/kythe/kythe/commit/d97ca443b265685c10e7caa7d4cc1c9f97933ee7))
+* **guava:**  allow using guava along with modules (#3559) ([78c92cb2](https://github.com/kythe/kythe/commit/78c92cb21d460072c6915d93f94e27867f93eeee))
```
Comment (schroederc) [inline RELEASES.md]: Not relevant for release.

File: RELEASES.md [hunk 14]
```diff
@@ -1,5 +1,321 @@
...
+  *  update release docs to specify kzip (#3618) ([5192f831](https://github.com/kythe/kythe/commit/5192f831574bdfde26c21c414343abb8ea70c45a))
+  *  update getting started for clang-3.6 (#3576) ([39b7473c](https://github.com/kythe/kythe/commit/39b7473cfb7d2fd925c00debe5c68edb241f309d))
+  *  clarify VName documentation in analysis.proto (#3560) ([44c348b3](https://github.com/kythe/kythe/commit/44c348b344ac3fea7ef4e3e53b645f424c11a796))
+* **extraction:**
+  *  update bazel-extractor Docker image to clang-6.0 (#4066) ([58f6250f](https://github.com/kythe/kythe/commit/58f6250f93df187418eac6bfeee970c02d844dc8))
+  *  Set cxx extractor kzip times to a constant (#4086) ([642c9330](https://github.com/kythe/kythe/commit/642c933026acafd44f49de81f12e9f21e4fb6de0))
+  *  Don't include time in kzips for java (#4085) ([3c25f1a0](https://github.com/kythe/kythe/commit/3c25f1a07109179ace64dc3517a49b7f4b07dbc0))
+  *  Don't include kzip creation time in kzip (#4083) ([5092d9d5](https://github.com/kythe/kythe/commit/5092d9d5a0a01131e29ad59dce791815a71b85ec))
+* **go:**
```
Comment (schroederc) [inline RELEASES.md]: Put into relevant categories below.

File: RELEASES.md [hunk 15]
```diff
@@ -1,5 +1,321 @@
...
+  *  use build_config when claiming implicit AST subtrees (#3619) ([8868db03](https://github.com/kythe/kythe/commit/8868db03a0d456ce93f2b22b09a4a82267d685b7))
+  *  attach build_config to file claim tokens (#3590) ([05800883](https://github.com/kythe/kythe/commit/058008830d4bc4cbb693c0146685d10fbd2a93a4))
+* **cxx_verifier:**  fix MSAN error on use-of-uninitialized (#4010) ([08fa1823](https://github.com/kythe/kythe/commit/08fa18231b1d52da3788471ad6f45e52720ccd54))
+* **dev:**
+  *  format shasum_tool output to match sha256sum (#3573) ([611d903a](https://github.com/kythe/kythe/commit/611d903a6a88b0acddf2875e84efd54b91d10c7e))
+  *  include brotli and leveldb in the macOS package list (#3519) ([48264e2b](https://github.com/kythe/kythe/commit/48264e2bd9ad29ec31ec300acd19920868b99267))
+  *  allow specification of a correct Bison version (#3516) ([a3f8334f](https://github.com/kythe/kythe/commit/a3f8334f75d605cfb037c31b17584fa6aa693edb))
+  *  use POSIX safe flags for tr (#3515) ([9dbb2a1a](https://github.com/kythe/kythe/commit/9dbb2a1ab7a6b7668d81effa543659f4be8bf11b))
+* **docs:**
```
Comment (schroederc) [inline RELEASES.md]: Not released.

File: RELEASES.md [hunk 16]
```diff
@@ -1,5 +1,321 @@
...
+  *  normalize windows paths (#3742) ([167786ac](https://github.com/kythe/kythe/commit/167786acf6f2d10e1e8f3923da0a1f4f59e59a66))
+  *  insist that compilation units have working directories (#3716) ([d3e059d1](https://github.com/kythe/kythe/commit/d3e059d1ecbb74ffe61f8d47ede71e2b244d5933))
+  *  always add . to the VFS (#3715) ([a8e95930](https://github.com/kythe/kythe/commit/a8e9593033ed63573948d047e3afc4e31f400d99))
+  *  use URI-friendly encoding for VName fields (#3705) ([f9a7e299](https://github.com/kythe/kythe/commit/f9a7e299c8f89316e8a09635b143023a1950d6d3))
+  *  use build_config when claiming implicit AST subtrees (#3619) ([8868db03](https://github.com/kythe/kythe/commit/8868db03a0d456ce93f2b22b09a4a82267d685b7))
+  *  attach build_config to file claim tokens (#3590) ([05800883](https://github.com/kythe/kythe/commit/058008830d4bc4cbb693c0146685d10fbd2a93a4))
+* **cxx_verifier:**  fix MSAN error on use-of-uninitialized (#4010) ([08fa1823](https://github.com/kythe/kythe/commit/08fa18231b1d52da3788471ad6f45e52720ccd54))
+* **dev:**
```
Comment (schroederc) [inline RELEASES.md]: Not relevant for release.

File: RELEASES.md [hunk 17]
```diff
@@ -1,5 +1,321 @@
...
+  *  Extraction exits with nonzero error code if unable to create kzip (#3629) ([2c553016](https://github.com/kythe/kythe/commit/2c5530168eea3977d1300db57fa458f50f6aad16))
+* **build:**
+  *  include C++14 options in default build flags (#4008) ([39d1ccbb](https://github.com/kythe/kythe/commit/39d1ccbbdd99a7a92a370676f6e8bdf912c53b1c), closes [#4006](https://github.com/kythe/kythe/issues/4006))
+  *  fully qualify workspace references in lexyacc rules (#3656) ([2f05e066](https://github.com/kythe/kythe/commit/2f05e066e21f2d4d055aadd34e790b36334e72c8))
+  *  remove duplicate src paths in java_extract_kzip (#3530) ([252c0466](https://github.com/kythe/kythe/commit/252c046619c21ce621ed049697cef3813bfb0276))
+* **compdb_test:**  ignore log files instead of treating as kzip (#3697) ([cdde2ae5](https://github.com/kythe/kythe/commit/cdde2ae52db523444ccdb632a4eef9dbc83195e2))
+* **cxx:**
+  *  better support cuda extractions (#3747) ([2d7f89c6](https://github.com/kythe/kythe/commit/2d7f89c69a1cc816d21c7d86faf6791d4afda8e7))
+  *  remove dependency on ncurses/terminfo (#3644) ([dfeaa897](https://github.com/kythe/kythe/commit/dfeaa897b79b28fa38a8603890abe3fe6af39a60))
```
Comment (schroederc) [inline RELEASES.md]: Not relevant for release?

File: RELEASES.md [hunk 18]
```diff
@@ -1,5 +1,321 @@
...
+  *  Extraction exits with nonzero error code if unable to create kzip (#3629) ([2c553016](https://github.com/kythe/kythe/commit/2c5530168eea3977d1300db57fa458f50f6aad16))
+* **build:**
+  *  include C++14 options in default build flags (#4008) ([39d1ccbb](https://github.com/kythe/kythe/commit/39d1ccbbdd99a7a92a370676f6e8bdf912c53b1c), closes [#4006](https://github.com/kythe/kythe/issues/4006))
+  *  fully qualify workspace references in lexyacc rules (#3656) ([2f05e066](https://github.com/kythe/kythe/commit/2f05e066e21f2d4d055aadd34e790b36334e72c8))
+  *  remove duplicate src paths in java_extract_kzip (#3530) ([252c0466](https://github.com/kythe/kythe/commit/252c046619c21ce621ed049697cef3813bfb0276))
+* **compdb_test:**  ignore log files instead of treating as kzip (#3697) ([cdde2ae5](https://github.com/kythe/kythe/commit/cdde2ae52db523444ccdb632a4eef9dbc83195e2))
+* **cxx:**
+  *  better support cuda extractions (#3747) ([2d7f89c6](https://github.com/kythe/kythe/commit/2d7f89c69a1cc816d21c7d86faf6791d4afda8e7))
```
Comment (schroederc) [inline RELEASES.md]: `cxx_extractor`

File: RELEASES.md [hunk 19]
```diff
@@ -1,5 +1,321 @@
...
+  *  include jsr250 jar for java extraction (#3647) ([f8dc4355](https://github.com/kythe/kythe/commit/f8dc43552164190f90ae00dfb687515b10243856))
+  *  add curses to always-installed packages in the Bazel dockerfile (#3642) ([2fe11a64](https://github.com/kythe/kythe/commit/2fe11a640861bf666f99c267e4c26bf27d2c6602))
+  *  Extraction exits with nonzero error code if unable to create kzip (#3629) ([2c553016](https://github.com/kythe/kythe/commit/2c5530168eea3977d1300db57fa458f50f6aad16))
+* **build:**
+  *  include C++14 options in default build flags (#4008) ([39d1ccbb](https://github.com/kythe/kythe/commit/39d1ccbbdd99a7a92a370676f6e8bdf912c53b1c), closes [#4006](https://github.com/kythe/kythe/issues/4006))
+  *  fully qualify workspace references in lexyacc rules (#3656) ([2f05e066](https://github.com/kythe/kythe/commit/2f05e066e21f2d4d055aadd34e790b36334e72c8))
+  *  remove duplicate src paths in java_extract_kzip (#3530) ([252c0466](https://github.com/kythe/kythe/commit/252c046619c21ce621ed049697cef3813bfb0276))
+* **compdb_test:**  ignore log files instead of treating as kzip (#3697) ([cdde2ae5](https://github.com/kythe/kythe/commit/cdde2ae52db523444ccdb632a4eef9dbc83195e2))
```
Comment (schroederc) [inline RELEASES.md]: Not relevant for release.

File: RELEASES.md [hunk 20]
```diff
@@ -1,5 +1,321 @@
...
+*   extract test files in TEST_TMPDIR instead of runfiles dir (#29) ([f94a1299](https://github.com/kythe/kythe/commit/f94a1299e12b25e9e363df194edcaa15f1bdfb45))
+*   get toolchains for remote builds working correctly (#7) ([b034a8f3](https://github.com/kythe/kythe/commit/b034a8f360f7279b8a0025e91a24fb394c5c7e91))
+* **api:**  filter decoration overrides by desired build configs (#3739) ([3f4ef51b](https://github.com/kythe/kythe/commit/3f4ef51b7ba9c347ebc7bca151d19a6bc55a5309))
+* **bazel:**
+  *  proto toolchain not guaranteed to support no_enforce_api_compatibility (#3693) ([12524aca](https://github.com/kythe/kythe/commit/12524aca167402992b601d04a4621017773c0246))
+  *  include jsr250 jar for java extraction (#3647) ([f8dc4355](https://github.com/kythe/kythe/commit/f8dc43552164190f90ae00dfb687515b10243856))
+  *  add curses to always-installed packages in the Bazel dockerfile (#3642) ([2fe11a64](https://github.com/kythe/kythe/commit/2fe11a640861bf666f99c267e4c26bf27d2c6602))
+  *  Extraction exits with nonzero error code if unable to create kzip (#3629) ([2c553016](https://github.com/kythe/kythe/commit/2c5530168eea3977d1300db57fa458f50f6aad16))
+* **build:**
```
Comment (schroederc) [inline RELEASES.md]: Not relevant for release.

File: RELEASES.md [hunk 21]
```diff
@@ -1,5 +1,321 @@
...
+*   downstream fix to provider syntax (#3508) ([752037dc](https://github.com/kythe/kythe/commit/752037dc9eed8bc0fdde7f68aed4dc7814acdd0b))
+*   go style errors introduced in previous commit (#3502) ([202187c5](https://github.com/kythe/kythe/commit/202187c59cd7f14d38bdc115a33e291c58be3738))
+*   propagating a downstream fix on typescript (#3496) ([04b7bb2d](https://github.com/kythe/kythe/commit/04b7bb2dc5ce7a8f8aca0bad16e4eefb28252404))
+*   address part of #3459 by claiming comments for annotation lines (#3489) ([bcddb97c](https://github.com/kythe/kythe/commit/bcddb97c57a97dcd0cc1580c5213f899a2d8230b))
+*   extract test files in TEST_TMPDIR instead of runfiles dir (#29) ([f94a1299](https://github.com/kythe/kythe/commit/f94a1299e12b25e9e363df194edcaa15f1bdfb45))
+*   get toolchains for remote builds working correctly (#7) ([b034a8f3](https://github.com/kythe/kythe/commit/b034a8f360f7279b8a0025e91a24fb394c5c7e91))
+* **api:**  filter decoration overrides by desired build configs (#3739) ([3f4ef51b](https://github.com/kythe/kythe/commit/3f4ef51b7ba9c347ebc7bca151d19a6bc55a5309))
+* **bazel:**
```
Comment (schroederc) [inline RELEASES.md]: Not relevant for release.

File: RELEASES.md (hunk 14 above)
Comment (salguarnieri) [reply RELEASES.md]: done

File: RELEASES.md (hunk 20 above)
Comment (salguarnieri) [reply RELEASES.md]: done

**Pull request merged**